examples/.ignore

```diff
@@ -2,3 +2,4 @@ mod.ts
 multisig_transfer.ts
 smart_contract.ts
 xcm_teleport_assets.ts
+examples.test.ts
```

Comment:
I'd rather this test not be in the examples folder.

Could you move it to `_tasks/test_examples.ts`?